pingdom/contact_types.go

```diff
@@ -0,0 +1,63 @@
+package pingdom
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// Contact represents a Pingdom contact
+type Contact struct {
+	Name               string `json:"name"`
+	Email              string `json:"email,omitempty"`
+	Cellphone          string `json:"cellphone,omitempty"`
+	Countryiso         string `json:"countryiso,omitempty"`
+	CountryCode        string `json:"countrycode,omitempty"`
+	Defaultsmsprovider string `json:"defaultsmsprovider,omitempty"`
+	Directtwitter      bool   `json:"directtwitter,omitempty"`
+	Twitteruser        string `json:"twitteruser,omitempty"`
+	Iphonetokens       string `json:"iphonetokens,omitempty"`
+	Androidtokens      string `json:"androidtokens,omitempty"`
```

Comment:
Looks great!  Would you mind updating some of the capitalization for a few of these fields to improve readability as well as for consistency with how we've capitalized other fields? I was thinking the following ones:
```
CountryISO
DefaultSMSProvider
DirectTwitter
TwitterUser
IphoneTokens
AndroidTokens
```